de.dlr.sc.virsat.model.concept/src/de/dlr/sc/virsat/model/concept/generator/ereference/ExternalGenModelHelper.xtend

```diff
@@ -0,0 +1,85 @@
+/*******************************************************************************
+ * Copyright (c) 2008-2019 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package de.dlr.sc.virsat.model.concept.generator.ereference
+
+import de.dlr.sc.virsat.model.dvlm.categories.propertydefinitions.EReferenceProperty
+import de.dlr.sc.virsat.model.dvlm.concepts.Concept
+import org.eclipse.emf.codegen.ecore.genmodel.GenModel
+import org.eclipse.emf.codegen.ecore.genmodel.GenPackage
+import org.eclipse.emf.common.util.URI
+import org.eclipse.emf.ecore.EPackage
+import org.eclipse.emf.ecore.resource.Resource
+import org.eclipse.emf.ecore.resource.ResourceSet
+
+class ExternalGenModelHelper {
+	
+	/**
+	 * Get the full qualified java class name of property
+	 */
+	def getEObjectClass(EReferenceProperty property) {

```

Comment:
Not sure, if this method should maybe placed into the CategoryHelper?

Reply:
I think we only need it in the concept language generator... This method returns the Java class of the referenced EClass. I don't think it's relevant in the CategoryHelper? To get the EClass model element we have the EReferencePropertyHelper. (Which is analog to the EnumPropertyHelper we already have)